src/odemis/gui/conf/file.py

```diff
@@ -541,4 +550,3 @@ def get_sh_calib(self, shid):
                 logging.exception("Failed to read calibration data")
 
         return None
-
```

Comment:
Pep8 is one empty line at the end of a file or not?

Reply:
yes, there were two in this file, github sort of hides the last empty line.

Reply:
Ah didn't know that it is hidden as I could not expand the file further.